modules/protobuf/metadata.json

```diff
@@ -3,7 +3,8 @@
     "maintainers": [],
     "versions": [
         "3.19.0",
-        "3.19.2"
+        "3.19.2",
```

Comment:
Should we yank the versions that are affected by the issue?

Reply:
The new version should be backwards compatible, so probably we should, will send a PR.